### ISSUE TITLE
Images with ampersand & cause problems

### ISSUE THREAD

**Issue by wonderly2** [opencart/opencart#9150]
I failed to update my store image because the filename had an ampersand in it. It appears to save, but when you go back the default image is restored. Also the image cannot be deleted from the Image Manager. It says "Warning: You can not delete this directory!"

**Comment by danielkerr** [comment]
use only a-Z AND 0-9 FOR file names

**Comment by wonderly2** [comment]
A note to that effect could help others avoid the headache I had yesterday. It took awhile to figure out... was it the image format? size? I was looking for some criteria to tell me what I was doing wrong. Figuring out it was the filename was an accident.
I've never contributed to someone else's project. Is that something I could add?

**Comment by osworx** [comment]
> 
> .. what I was doing wrong. Figuring out it was the filename was an accident.
>

Well, is should be common knowledge these days that folders, files (therefore also image names)  **should NOT have any special characters like !"§$%&/()=?*'_:; AND spaces AND Umlauts äöüÄÖÜ**

As Daniel already replied, use ONLY latin characters a-z and/or A-Z and/or 0-9 as well as - or _

**Comment by wonderly2** [comment]
I understand the restriction and it will no longer be an issue for me. And I'm not complaining, at all. I high appreciate the many many open source projects out there which benefit me. I was just thinking I should try and help improve some if possible. I wouldn't think it would be desirable for a file with standard Windows filename characters to break the application and cause a user such grief. The "K_&_K_Logo.jpg" filename did not cause me trouble anywhere else; including when I uploaded to my personal PHP applications. I've been coding for a decade and it wasn't obvious to me. But oh well, I tried. Thanks guys for all your work!
